lib/command.c

```diff
@@ -1454,7 +1454,7 @@ DEFUN (config_help,
        "Description of the interactive help system\n")
 {
 	vty_out(vty,
-		"Quagga VTY provides advanced help feature.  When you need help,\n\
+		"FRR VTY provides advanced help feature.  When you need help,\n\
```

Comment:
nit: "features" , or "a help feature" ?

Reply:
dunno I didn't write that part.  I was just interested in the Quagga -> FRR